### PR TITLE
Ymes 3 use billing name on guest checkout

### PR DESCRIPTION
**Pull request by haroon-chaudhry**

### CODE REVIEW

File: app/code/community/Yotpo/Yotpo/Model/Mail/Observer.php [hunk 1]
```diff
@@ -39,9 +39,12 @@ public function mail_after_purchase($observer)
 			if (!$order->getCustomerIsGuest()) {
 				$data["user_reference"] = $order->getCustomerId();
 			}
-
+			$customer = $order->getCustomerName();
```
Comment (omryYotpo) [inline app/code/community/Yotpo/Yotpo/Model/Mail/Observer.php]: change naming to customer_name

File: app/code/community/Yotpo/Yotpo/Model/Mail/Observer.php [hunk 2]
```diff
@@ -39,9 +39,12 @@ public function mail_after_purchase($observer)
...
+			if($order->getCustomerIsGuest()){ // If checkout is guest checkout, than load billing name instead.
```
Comment (omryYotpo) [inline app/code/community/Yotpo/Yotpo/Model/Mail/Observer.php]: please remove comment

**Comment by haroon-chaudhry** [comment]
@FarhanIslamIDT could you please fix review comments ASAP.

**Comment by FarhanIslamIDT** [comment]
Done the needful changes in 70821454fcd68776b04436310778a8b77f3d10fc

**Pull request merged**